blivet/devices/btrfs.py

```diff
@@ -494,11 +494,18 @@ def _set_format(self, fmt):
 
         # propagate mount options specified for members via kickstart
         opts = "subvol=%s" % self.name
+        has_compress = False
         if self.volume.format.mountopts:
             for opt in self.volume.format.mountopts.split(","):
                 # do not add members subvol spec
                 if not opt.startswith("subvol"):
                     opts += ",%s" % opt
+                if opt.startswith("compress"):
+                    has_compress = True
+
+        # add default compression settings
+        if blivet.flags.btrfs_compression and not has_compress:
```

Comment:
Pylint complains about `blivet` not being defined, `flags` are already imported here, so use just `flags.btrfs_compression`, thanks.

```
E0602(undefined-variable):blivet/devices/btrfs.py:507,11: BTRFSSubVolumeDevice._set_format: Undefined variable 'blivet'
E0602(undefined-variable):blivet/devices/btrfs.py:508,37: BTRFSSubVolumeDevice._set_format: Undefined variable 'blivet'
```

Reply:
yeah, I fixed that locally but apparently didn't `git commit --amend -a` so pushed the wrong version. Silly me.